recipes/qpdf/all/conanfile.py

```diff
@@ -119,12 +124,24 @@ def generate(self):
 
     def _patch_sources(self):
         apply_conandata_patches(self)
+        # we generally expect to have one crypto in-place, but need to patch the found mechanics
+        # since we avoid currently the correct pkg_config
+        replace_in_file(self, os.path.join(self.source_folder, "libqpdf", "CMakeLists.txt"),
```

Comment:
This can go in the parch I think

Reply:
Mh, it is possible, but i would like to have the handling for the FOUND_CRYPTO, USE_CRYPTO_OPENSSL, USE_CRYPTO_GNUTLS, USE_CRYPTO_NATIVE next to each other, since they are already contextwise near to each other.